Clients/Xamarin.Interactive.Client/Workbook/Models/CodeCellBuffer.cs

```diff
@@ -1,69 +0,0 @@
-//
-// Author:
-//   Aaron Bockover <abock@xamarin.com>
-//
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Text;
-
-using Microsoft.CodeAnalysis.Text;
-
-namespace Xamarin.Interactive.Workbook.Models
-{
-    sealed class CodeCellBuffer : SourceTextContainer, ICellBuffer
```

Comment:
Did you mean for this deletion to end up in this commit?

Reply:
Does it matter? A lot of these were pretty mixed up and hard to separate.

Reply:
I mean, no, it doesn't look like that should have been in that commit. Meh.